pages/common/mtr.md

```diff
@@ -22,3 +22,7 @@
 - Wait for a given time (in seconds) before sending another packet to the same hop:
 
 `mtr -i {{seconds}} {{host}}`
+
+- Perform a reliable package drop test that can be shown to the Internet Service Provider (ISP):
```

Comment:
This adds a cheat-sheet like command which combines several flags to achieve a result. Usually the way tldr pages are written in a way where each flag is explained in a separate example allowing users to build up more complex examples themselves.

I see 4 different flags used here but I don't see a clear explanation of each of them in the description.

It might be difficult to accomodate such an example in a tldr page.

Reply:
What do you propose then?

Reply:
One thing to note is that we limit the number of examples to 8 per page. So we restrict ourselves to only the most important commands one will use on a day to day basis. If you believe these flags are common enough for an average user, I would suggest to break up this cheat-sheet like approach into explaining what the command actually does, by explaining each flag in a separate example.